### ISSUE TITLE
getJails() undefined for type Jails

### ISSUE THREAD

**Issue by DrewAMitchell** [EssentialsX/Essentials#1758]
For development environment, I can't tell if this issue is just specific to me or if it's just an issue that Essentials is able to function with.

https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/com/earth2me/essentials/Jails.java
In [EssentialsUpgrade.java](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java#L354), I get an error saying "getJails() is undefined for type Jails."

The interface that Jails implements doesn't have getJails() as an abstract method, and the class itself doesn't... is this just my IDE being weird?
  
@SupaHam

**Comment by caojohnny** [comment]
Add the lombok plugin to your IDE or run `mvn clean install`.

**Comment by DrewAMitchell** [comment]
@AgentTroll I clean installed and added lombok to the build path (Eclipse), it's still the only error in my project.

**Comment by caojohnny** [comment]
It's not an Essentials issue, that's for sure. You should check the lombok docs since I am not familiar with eclipse integration and lombok.

**Comment by mdcfe** [comment]
Closing as this doesn't appear to actually affect EssentialsX.

**Comment by DrewAMitchell** [comment]
@AgentTroll @md678685 After doing a reasonable amount of Googling, I figured out that my perceived issue was lombok, and that I hadn't understood fully what @AgentTroll had asked. I didn't realize that lombok made modifications to the development environment. After running java -jar lombok.jar, I was able to close/reopen Eclipse, and the errors had disappeared. I guess lombok is so smart that it modifies the compilation problems that one running vanilla Eclipse would encounter with EssentialsX. Regardless, after running java -jar lombok.jar as administrator, and rebooting my IDE, I Maven-Updated the project and voila! They're gone. Thanks @AgentTroll!
My hope is that this will now be a good Google result for anyone who doesn't use lombok often like myself.

**Comment by mdcfe** [comment]
Not sure what you mean - you should just install the [relevant IDE plugin](https://projectlombok.org/setup/eclipse) and use Maven to build instead of your IDE.

**Comment by DrewAMitchell** [comment]
@md678685 Well, that's what I'm trying to say, my bad. I ran lombok.jar through the command line because why not? But it's the same as clicking the jar file and running that GUI. I'm using Maven to build, I had just misinterpreted "add the lombok plugin to your IDE" as a build path request since - once again, I've never worked with Lombok before and didn't know it modified the integrated development environment behavior.
